packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -935,6 +938,97 @@ export const RemoteServerFactory = {
     }
 }
 
+/**
+ * Patches `res.setHeader` to ensure that the Content-Security-Policy header always includes the
+ * directives required for PWA Kit to work.
+ * @param {express.Request} req Express request object
+ * @param {express.Response} res Express response object
+ * @param {express.NextFunction} next Express next callback
+ */
+export const enforceSecurityHeaders = (req, res, next) => {
+    /** CSP-compatible origin for Runtime Admin. */
+    // localhost doesn't include a protocol because different browsers behave differently :\
+    const runtimeAdmin = isRemote() ? 'https://runtime.commercecloud.com' : 'localhost:*'
+    /**
+     * Map of directive names/values that are required for PWA Kit to work.
+     * @type Object.<string, string[]>
+     */
+    const directives = {
+        'connect-src': ["'self'", 'api.cquotient.com', runtimeAdmin],
+        'frame-ancestors': [runtimeAdmin],
+        'img-src': ["'self'", '*.commercecloud.salesforce.com', 'data:'],
```

Comment:
Same question goes for '*.commercecloud.salesforce.com'? Template specific or SDK global.